Q2/answer.txt

```diff
@@ -0,0 +1,5 @@
+- When we attach the script in the head tag, the browser won't be able to find any element whose id matches with the given id as the element itself doesn't exist when the browser will be executing the script.So, it will just show Hello World! in the screen.
+
+- When we attach the script at the start of the body tag, again the browser won't find the element because the element doesn't exist when the browser will be executing the script. So, it will just show Hello World! in the screen.
```

Comment:
Also it will give error as its trying to access .innerHtml property on undefine (as the element was not found)